client/analysisui/levelselector.js

```diff
@@ -37,13 +38,18 @@ const LevelSelector = function(params) {
         }
 
         let t = '<select class="silky-option-input silky-option-combo-input jmv-level-selector silky-control-margin-' + this.getPropertyValue('margin') + '">';
+        if (this.getPropertyValue('allowNone'))
+            t += '<option>None</option>';
```

Comment:
should we put none in square brackets to make it clear it's a special value? i.e. `[None]`?

Reply:
should this be a property of the option in the .a.yaml? i think it needs to be so the R knows what's going on.